lib/babelish/csv2android.rb

```diff
@@ -13,7 +13,8 @@ def language_filepaths(language)
       require 'pathname'
       output_name = "strings.xml"
       output_name = "#{@output_basename}.xml" unless @output_basename.empty?
-      filepath = Pathname.new(@file_path) + "values-#{language.code}" + output_name
+      region = language.region.to_s.empty? ? "" : "-#{language.region}"
```

Comment:
@beloso isn't it `region = language.region.to_s.empty? ? "" : "-r#{language.region}"` ?
missing `r`